Mathlib/MeasureTheory/Integral/SetToL1.lean

```diff
@@ -320,7 +321,7 @@ theorem map_setToSimpleFunc (T : Set α → F →L[ℝ] F') (h_add : FinMeasAddi
     refine hfp i hi.1 fun hi0 => ?_
     rw [hi0, hg] at hi
     exact h0 hi.2.symm
-  · intro i _j hi _ hij
+  · intro i _ j _ hij
```

Comment:
I'll trust you that the variables are well-named: didn't verify.

Reply:
Before:
```
i : G
_j : i ∈ filter (fun b ↦ g b = g (f a)) f.range
hi : G
a✝ : hi ∈ filter (fun b ↦ g b = g (f a)) f.range
hij : i ≠ hi
```

After:
```
i : G
a✝¹ : i ∈ filter (fun b ↦ g b = g (f a)) f.range
j : G
a✝ : j ∈ filter (fun b ↦ g b = g (f a)) f.range
hij : i ≠ j
```